tests/std/tests/GH_000625_vector_bool_optimization/test.cpp

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <algorithm>
+#include <assert.h>
+#include <vector>
+
+using namespace std;
+
+constexpr int blockSize = 32;
+static_assert(blockSize == _VBITS, "Invalid block size");
+
+void test_fill_helper(const size_t length) {
+    // No offset
+    {
+        vector<bool> result_true(length, true);
+        result_true.resize(length + 3, false);
+        vector<bool> dest_true(length + 3, false);
+        fill(dest_true.cbegin(), prev(dest_true.cend(), 3), true);

```

Comment:
And it appears that this test won't compile because it's attempting to fill through `cbegin`.

"And if you gaze long into a `const` qualifier, the `const` qualifier also gazes into you." - Friedrich Nietzsche

Reply:
Done